ArmNavigation/rrt_star_seven_joint_arm_control/rrt_star_seven_joint_arm_control.py

```diff
@@ -374,31 +374,38 @@ def main():
     path = rrt_star.planning(animation=show_animation,
                              search_until_max_iter=False)
 
-    if path is None:
-        print("Cannot find path")
-    else:
-        print("found path!!")
+    plt.ion()
+    while True:
+        if path is None:
+            print("Cannot find path")
+        else:
+            print("Found path!!")
 
-        # Draw final path
-        if show_animation:
-            ax = rrt_star.draw_graph()
-
-            # Plot final configuration
-            x_points, y_points, z_points = seven_joint_arm.get_points(path[-1])
-            ax.plot([x for x in x_points],
-                    [y for y in y_points],
-                    [z for z in z_points],
-                    "o-", color="red", ms=5, mew=0.5)
+            # Draw final path
+            if show_animation:
+                ax = rrt_star.draw_graph()
 
-            for i, q in enumerate(path):
-                x_points, y_points, z_points = seven_joint_arm.get_points(q)
+                # Plot final configuration
+                x_points, y_points, z_points = seven_joint_arm.get_points(path[-1])
                 ax.plot([x for x in x_points],
                         [y for y in y_points],
                         [z for z in z_points],
-                        "o-", color="grey",  ms=4, mew=0.5)
-                plt.pause(0.01)
-
-            plt.show()
+                        "o-", color="red", ms=5, mew=0.5)
+
+                for q in path:
+                    x_points, y_points, z_points = seven_joint_arm.get_points(q)
+                    ax.plot([x for x in x_points],
+                            [y for y in y_points],
+                            [z for z in z_points],
+                            "o-", color="grey",  ms=4, mew=0.5)
+                    plt.pause(0.1)
```

Comment:
increasing the pause time, because it seems way to fast

Reply:
Increasing the pause time makes sense for me, but what is for other changes?